docs/marketplace/dev/feature-integration-guides/202108.0/merchant-portal-marketplace-merchant-feature-integration.md

```diff
@@ -37,28 +37,3 @@ Make sure that the following modules have been installed:
 | MerchantProfileMerchantPortalGui | vendor/spryker/merchant-profile-merchant-portal-gui |
 
 {% endinfo_block %}
-
-### 2) Set up transfer objects
-
-Generate transfer changes:
-
-```bash
-console transfer:generate
-```
-{% info_block warningBox "Verification" %}
-
-Make sure that the following changes have been applied in transfer objects:
-
-| TRANSFER  | TYPE  | EVENT | PATH  |
-| ------------- | ------ | ------- | ----------------- |
-| MerchantProfile | object | Created | src/Generated/Shared/Transfer/MerchantProfile |
-
-{% endinfo_block %}
-
-### Zed translations
-
-Generate a new translation cache for Zed:
-
-```bash
-console translator:generate-cache
-```
```

Comment:
why was it removed? the modules contains some new translations

Reply:
brought it back